cmd/operator-sdk/new/cmd.go

```diff
@@ -459,18 +458,16 @@ func getDeps() error {
 }
 
 func initGit() error {
-	if skipGit {
+	if !gitInit {
```

Comment:
Nit: Should we move this check such that we only call `initGit` if `gitInit` is `true`

Reply:
Yeah that's probably better. We do that with the `skipValidation` flag variable as well.